### PR TITLE
[v6r15] Fix log messages pilot (MJF specifically)

### PR DESCRIPTION
**Pull request by fstagni**
Plus: 
- fixed bug in MJF (wrong indent)
- added comprehensive integration test

### CODE REVIEW

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py [hunk 1]
```diff
@@ -776,20 +773,25 @@ def execute( self ):
...
+    retCode, cpuTimeOutput = self.executeAndGetOutput( 'dirac-wms-get-queue-cpu-time %s %s' % ( configFileArg,
+                                                                                                self.pp.localConfigFile ),
+                                                       self.pp.installEnv )
+
     if retCode:
       self.log.error( "Failed to determine cpu time left in the queue [ERROR %d]" % retCode )
       self.exitWithError( retCode )
+
+    for line in cpuTimeOutput.split( '\n' ):
+      if re.search( "CPU time left determined is *", line ):
```
Comment (phicharp) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: Is regular expression really needed?

File: WorkloadManagementSystem/scripts/dirac-wms-get-queue-cpu-time.py [hunk 2]
```diff
@@ -28,5 +28,5 @@
   from DIRAC.WorkloadManagementSystem.Client.CPUNormalization import getCPUTime
   cpuTime = getCPUTime( CPUNormalizationFactor )
   # I hate this kind of output... PhC
-  print cpuTime
+  print "CPU time left determined is", cpuTime
```
Comment (phicharp) [inline WorkloadManagementSystem/scripts/dirac-wms-get-queue-cpu-time.py]: "determined as" ?

**Comment by phicharp** [comment]
Review OK (2 minor remarks)

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py (hunk 1 above)
Comment (fstagni) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: I think it's the simplest solution.

Comment (sposs) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: why not 
```
if line.find("CPU time left determined is")>-1 
```

Comment (chaen) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: ```if "CPU time left determined is" in line``` ?

File: WorkloadManagementSystem/Client/CPUNormalization.py [hunk 3]
```diff
@@ -185,8 +185,21 @@ def getCPUNormalization( reference = 'HS06', iterations = 1 ):
 
 
 def getCPUTime( cpuNormalizationFactor ):
-  """ Trying to get CPUTime (in seconds) from the CS. The default is a large 9999999, that we may consider as "Infinite".
+  """ Trying to get CPUTime left for execution (in seconds).
+
+      It will first look to get the work left looking for batch system information useing the TimeLeft utility.
+      If it suceeds, it will convert it in real second, and return it.
```
Comment (chaen) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: minor but typo in 'succeeds'

File: WorkloadManagementSystem/Client/CPUNormalization.py [hunk 4]
```diff
@@ -185,8 +185,21 @@ def getCPUNormalization( reference = 'HS06', iterations = 1 ):
...
+
+      If it fails, it tries to get it from the static info found in CS.
+      If it fails, it returns the default, which is a large 9999999, that we may consider as "Infinite".
```
Comment (chaen) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: we could use ```sys.maxint``` for all these 'infinite' values. I mean, not only here, but throughout DIRAC

File: tests/Integration/WorkloadManagementSystem/Test_TimeLeft.sh [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+
+###############################################################################
+# Can't find anywhere a batch plugin, not even MJF
+
+python $DIRACROOT/WorkloadManagementSystem/scripts/dirac-wms-get-queue-cpu-time.py $DIRACROOT/tests/Integration/WorkloadManagementSystem/pilot.cfg $DEBUG
+
+if [ $? -eq 0 ]
+then
+  echo "\nSuccess\n\n"
```
Comment (chaen) [inline tests/Integration/WorkloadManagementSystem/Test_TimeLeft.sh]: If you want the '\n' to be interpreted, you need the -e flag

File: WorkloadManagementSystem/Client/CPUNormalization.py (hunk 4 above)
Comment (fstagni) [inline WorkloadManagementSystem/Client/CPUNormalization.py]: Good idea, but in another PR

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py (hunk 1 above)
Comment (sposs) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: indeed, even better

Comment (fstagni) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: Ah yes. Changing it.

**Comment by chaen** [comment]
Review OK. Just to note that here Landscape did not run again

**Pull request merged**